app/dialogs.js

```diff
@@ -140,9 +155,14 @@ var appDialogs = {
         dialog.showMessageBox(win, {
             title: "Images Copied",
             message: "Document images have been copied in the '" + path + "' directory.",
-            buttons: ['OK'],
+            buttons: ["OK"],
             noLink: true
         });
+    },
+
+    warnFileDeleted: function (path, callback) {
```

Comment:
I would rather display a warning here: "The file [path] doesn't exist anymore. Keep this file in editor?" => Yes/No
"No" closes the file, "Yes" sets the doc dirty and keep it open.